DigitalLearningSolutions.Web.Tests/Controllers/LearningPortal/LearningPortalControllerTests.cs

```diff
@@ -44,13 +46,21 @@ public void SetUp()
             config = A.Fake<IConfiguration>();
             filteredApiHelperService = A.Fake<IFilteredApiHelperService>();
 
+            var connection = ServiceTestHelper.GetDatabaseConnection();
+            learningLogItemsTestHelper = new LearningLogItemsTestHelper(connection);

```

Comment:
This is why the Github build is failing, we're doing some database stuff for some reason? We shouldn't need to do any database stuff to test things in the Web project

Reply:
removed and it is passing now!